config_scheduler.go

```diff
@@ -12,7 +12,7 @@ import (
 type SchedulerConfig struct {
 	TaskFinder                    string  `bson:"task_finder" json:"task_finder" yaml:"task_finder"`
 	HostAllocator                 string  `bson:"host_allocator" json:"host_allocator" yaml:"host_allocator"`
-	FreeHostFraction              float64 `bson:"free_host_fraction" json:"free_host_fraction" yaml:"free_host_fraction"`
+	FutureHostPercent             float64 `bson:"free_host_fraction" json:"free_host_fraction" yaml:"free_host_fraction"`
```

Comment:
It's fine to change the meaning of this, but we'll need a way to gracefully switch from the old value. Right now you need to make both changes immediately, otherwise the scheduler will do the wrong thing

Reply:
I think all we need to do is remember to modify the admin page value to be 40 instead of .4 when we deploy

Reply:
Since this setting is fetched every time the scheduler runs, we won't be able to do that

Reply:
https://github.com/evergreen-ci/evergreen/pull/4243/files#diff-ea89eb67522531ea4bc63a4e741ba227634589e5483a4787afa90c8147e14aefR100

Could this take care of it?

Reply:
No, because that requires you to unset the current value before deploying. Doing so will not pass the app validation, but even if you force a value in there it will be used immediately (but will be assumed to be a decimal 0<n<1)

Reply:
I've decided to dispense with the 0-100 stuff and just use a float

Reply:
thanks for pointing this out!